users/signals.py

```diff
@@ -0,0 +1,10 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from art_web_app.models import AuthUserModel
+from .models import Profile
+
+
+@receiver(post_save, sender=AuthUserModel)
+def update_profile_signal(instance, created, **kwargs):
```

Comment:
Just remind me...why do you need do update the profile when the user is CREATED?